infra/terraform/base_stacks/vpc.tf

```diff
@@ -14,7 +14,9 @@ resource "aws_cloudformation_stack" "vpc" {
     DynamoDBApiEnabled       = "Yes"
     ExecuteApiGatewayEnabled = "Yes"
     KMSApiEnabled            = "Yes"
+    S3ApiEnabled             = "Yes"
     SQSApiEnabled            = "Yes"
     SSMApiEnabled            = "Yes"
+    AllowRules               = "pass tls $HOME_NET any -> $EXTERNAL_NET 443 (tls.sni; content:\".account.gov.uk\"; endswith; msg:\"Pass TLS to *.account.gov.uk\"; flow:established; sid:2001; rev:1;)"
```

Comment:
I've just read up on this - ignore me

Reply:
I wonder if we don't need `AllowedDomains` (for now, at least) because of this:

When adding a domain to AllowedDomains, the following domains will automatically be included in the allow list for DNS lookups as well as enabling TLS 443 network firewall egress for them.
 - "*.account.gov.uk"   # any domain under .account.gov.uk

Reply:
The template is weird, if you provide `AllowedDomains` or `AllowRules`, regardless of what you provide then you enable the `ProvisionOutboundInternet` condition. 

This will add `.account.gov.uk` to the firwall domain list ([HERE](https://github.com/govuk-one-login/devplatform-deploy/blob/7d54730133ee56a35eec0eccfe2a7fed4376a260/vpc/template.yaml#L1835C1-L1848C72) 

and also create the firewall rule group which by default allows traffic from `.account.gov.uk` [HERE](https://github.com/govuk-one-login/devplatform-deploy/blob/7d54730133ee56a35eec0eccfe2a7fed4376a260/vpc/template.yaml#L3219C1-L3224C1) 

It is true you don't need `AllowedDomains` as in that purple box, but you could actually get the same effect by providing an `AllowedDomains` and no set the `AllowRules`.

Reply:
One could, for example, provide this: 

```suggestion
    AllowRules               = "# Value to enable Outbound Internet and the default rules"
```

Reply:
So we need either `AllowedDomains` or `AllowRules`, but defining only `AllowRules` is ok? 

Regarding the suggestion, I can change to that if that's more appropriate, but I quite like that the original one is very clear about what we are giving access to.

Reply:
happy to follow your lead here 👍

Reply:
I think AllowedDomains personally has a clearer syntax and is probably more understandable at a junior level of people looking at this for the first time.

Your call Ana - I'll approve whichever approach you go with.

Reply:
I prefer to keep it as it is.